examples/sample_measurement/pyproject.toml

```diff
@@ -12,9 +12,16 @@ click = ">=7.1.2, !=8.1.4" # mypy fails with click 8.1.4: https://github.com/pal
 [tool.poetry.group.dev.dependencies]
 mypy = ">=1.0"
 grpc-stubs = "^1.53"
+types-protobuf = "^4.21"
 # Uncomment to use prerelease dependencies.
-# ni-measurementlink-service = {path = "../..", develop = true}
+ni-measurementlink-service = {path = "../..", develop = true}
 
 [build-system]
 requires = ["poetry-core>=1.2.0"]
 build-backend = "poetry.core.masonry.api"
+
+[[tool.mypy.overrides]]
+module = [
+    "ni-measurementlink-service.*",
+]
+ignore_missing_imports = true
```

Comment:
Why was this necessary?

Reply:
The ignore_missing_imports, that is.

Reply:
I had copied this from the other examples' `pyproject.toml` and thought it is intended to solve the missing imports mypy issue, but everything still seems to work after I got rid of it so I've removed it.

Reply:
`ni-measurementlink-service` should not be missing.

Reply:
It should have ignored missing imports for `examples.*`